### PR TITLE
Draft post for poster comp announcement

### PR DESCRIPTION
**Pull request by weaverbel**
Not to be merged till after website post is out and not till after review - I will advise when ready

### CODE REVIEW

**Comment by fpsom** [comment]
LGTM - thanks @weaverbel !

Also as an extra incentive, would it make sense to add some Carpentries swag in addition to the free registration? I would expect (although I definitely do not have all the information) that this would be a relatively small additional cost, but (for me at least) it can be something tangible to "show off".

File: _posts/2017/12/2017-12-08-cc-poster-comp.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+The Carpentries are excited to announce our poster design competition where some lucky member of our community will win a **free** registration for [CarpentryCon 2018](https://www.carpentrycon.org/).
+
+Are there any artists out there who could design a fantastic poster to promote CarpentryCon 2018? If so, we want to see your design!
+
+The winning designer will gain free registration to CarpentryCon 2018. This includes registration for the conference only. It does not include travel to and from Dublin.
+
+#### Information for the Poster
+
+CarpentryCon 2018 will take place in Dublin from 30 May - 1 June, 2018 at [University College Dublin](http://www.ucd.ie/) with the overarching theme of *Building Globally, Connecting Locally*.
+
```
Comment (tracykteal) [inline _posts/2017/12/2017-12-08-cc-poster-comp.md]: I think it's 'Building Locally, Connecting Globally' ?

**Comment by tracykteal** [comment]
Carpentries swag would definitely be good to include for the winner.

File: _posts/2017/12/2017-12-08-cc-poster-comp.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<tr>
+<td valign="top" width="30%">What does the poster need to include?</td>
+<td valign="top">Posters should reflect the conference theme of <em>Building Globally, Connecting Locally</em>, and should also include the venue and dates (see above) and a link to the <a href="http://www.carpentrycon.org/">CarpentryCon website</a>.
+</td>
+</tr>
+
+<tr>
+<td valign="top" width="30%">How will the judging be done?</td>
+<td valign="top">Entries in the poster competition will be assessed by the members of the CarpentryCon task force who will choose the poster that best embodies the conference theme of <em>Building Globally, Connecting Locally</em>. The winner will be notified by email once the decision has been reached, and unsuccessful entrants will also be notified by email of the result. </td>
+</tr>
```
Comment (tracykteal) [inline _posts/2017/12/2017-12-08-cc-poster-comp.md]: Same here about the theme. Need to decide if it's this or "Building Locally, Connecting Globally"

**Comment by weaverbel** [comment]
Have fixed theme and will add swag. I have also created this page (still in my other fork) for the CC website: https://weaverbel.github.io/cc-draft/#comp - I can bring this over any time to the live CC site

**Pull request merged**